v2.1/online-schema-changes.md

```diff
@@ -0,0 +1,205 @@
+ERROR:  relation "foo_idx" does not exist
+ERROR:  current transaction is aborted, commands ignored until end of transaction block
+ROLLBACK
+~~~
+
+Add a column and then add a constraint against that column inside a user-level transaction:
+
+{% include copy-clipboard.html %}
+~~~ sql
+CREATE TABLE foo ();
+BEGIN;
+SAVEPOINT cockroach_restart;
+ALTER TABLE foo ADD COLUMN bar VARCHAR;
+ALTER TABLE foo ADD CONSTRAINT bar CHECK (foo IN ('a', 'b', 'c', 'd'));
+RELEASE SAVEPOINT cockroach_restart;
+COMMIT;
+~~~
+
+~~~
+CREATE TABLE
+BEGIN
+SAVEPOINT
+ALTER TABLE
+ERROR:  column "foo" not found for constraint "foo"
+ERROR:  current transaction is aborted, commands ignored until end of transaction block
+ROLLBACK
+~~~
+
+Add a column and then select against that column inside a user-level transaction:
+
+{% include copy-clipboard.html %}
+~~~ sql
+CREATE TABLE foo ();
+BEGIN;
+SAVEPOINT cockroach_restart;
+ALTER TABLE foo ADD COLUMN bar VARCHAR;
+SELECT bar FROM foo;
+RELEASE SAVEPOINT cockroach_restart;
+COMMIT;
+~~~
+
+~~~
+CREATE TABLE
+BEGIN
+SAVEPOINT
+ALTER TABLE
+ERROR:  column name "bar" not found
+ERROR:  current transaction is aborted, commands ignored until end of transaction block
+ROLLBACK
+~~~
+
+{{site.data.alerts.callout_success}}
+As of version 2.1, you *can* run schema changes inside the same transaction as a [`CREATE TABLE`][create-table] statement. For more information, [see this example](#run-schema-changes-inside-a-transaction-with-create-table).
+{{site.data.alerts.end}}
+
+## Examples
+
+### Run schema changes inside a transaction with `CREATE TABLE`
+
+As noted above, and in the list of [known limitations](known-limitations.html#schema-changes-within-transactions), you cannot run schema changes inside user transactions in general.
+
+However, as of version 2.1, you can run schema changes inside the same transaction as a [`CREATE TABLE`][create-table] statement. For example:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> BEGIN;
+  SAVEPOINT cockroach_restart;
+  CREATE TABLE fruits (
+        id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+        name STRING,
+        color STRING
+    );
+  INSERT INTO fruits (name, color) VALUES ('apple', 'red');
+  ALTER TABLE fruits ADD COLUMN inventory_count INTEGER DEFAULT 5;
+  ALTER TABLE fruits ADD CONSTRAINT name CHECK (name IN ('apple', 'banana', 'orange'));
+  SELECT name, color, inventory_count FROM fruits;
+  RELEASE SAVEPOINT cockroach_restart;
+  COMMIT;
+~~~
+
+The transaction succeeds with the following output:
+
+~~~
+BEGIN
+SAVEPOINT
+CREATE TABLE
+INSERT 0 1
+ALTER TABLE
+ALTER TABLE
++-------+-------+-----------------+
+| name  | color | inventory_count |
++-------+-------+-----------------+
+| apple | red   |               5 |
++-------+-------+-----------------+
+(1 row)
+COMMIT
+COMMIT
+~~~
+
+### Show all schema change jobs
+
+You can check on the status of the schema change jobs on your system at any time using the [`SHOW JOBS`][show-jobs] statement as shown below.
```

Comment:
I'd cut `as shown below` and end the sentence with the colon.

Reply:
Fixed.